filo/src/components/Navbar.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import { AppBar, Toolbar, List, ListItem, ListItemText, TextareaAutosize } from "@material-ui/core"
+import { makeStyles } from "@material-ui/core/styles"
+
+const useStyles = makeStyles({
+    navDisplayFlex: {
+        display: 'flex',
+        overflow: 'auto',
+        justifyContent: `space-between`
+    },
+    linkText: {
+        textDecoration: `none`,
+        textTransform: `uppercase`,
+        color: `white`
+    }
+});
+
+const navLinks = [
+    { title: `Ruleta`, path: `/Containers/Roulette/index.js` },
+    { title: `Restaurantes`, path: `/Containers/RestaurantDetail/index.js` },
+]
+
+const Navbar = () => {
+    const classes = useStyles();
+    return (
+        <AppBar position="static">
+            <Toolbar>
+                <List component="nav"
+                    aria-labelledby="main navigation"
+                    className={classes.navDisplayFlex}>
+                    {navLinks.map(({ title, path }) => (
+                        <a href={path} key={title} className={classes.linkText}>
```

Comment:
This is an SPA, we don't use anchor links in this project. Please use react router.